### ISSUE TITLE
toolBarHidden is using visibility:hidden, should be using display:none

### ISSUE THREAD

**Issue by sontek** [jpuri/react-draft-wysiwyg#592]
With visibility hidden it still takes up space:

<img width="350" alt="screenshot 2018-01-26 11 49 01" src="https://user-images.githubusercontent.com/151924/35453201-f8635bb4-028e-11e8-994f-5e4b9cfd963a.png">

**Comment by pfarnach** [comment]
Was just noticing the same thing.  Is there any reason to have `visibility` set to `hidden`?  It would make sense to me that the toolbar doesn't take up any space and that it's not even in the DOM if it's set to hidden.

**Comment by edencorbin** [comment]
I would agree with this, as a workaround I have to add negative margin when blurred.

**Comment by imoris11** [comment]
After a couple attempts, a work-around to this is by adding a class to the toolbar with a display of none.

`.hide-toolbar { display:none !important; }
`
And then
`toolbarClassName='hide-toolbar'`

Hope this helps.
